src/test/resources/processor/UploadModelRequestBody.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "all-MiniLM-L6-v2",
+  "version": 1,
+  "model_format": "TORCH_SCRIPT",
+  "model_task_type": "text_embedding",
+  "model_config": {
+    "model_type": "bert",
+    "embedding_dimension": 384,
+    "framework_type": "sentence_transformers",
+    "all_config": "{\"architectures\":[\"BertModel\"],\"max_position_embeddings\":512,\"model_type\":\"bert\",\"num_attention_heads\":12,\"num_hidden_layers\":6}"
+  },
+  "url": "https://api.quip-amazon.com/2/blob/MdZ9AAsfqat/y-6nBQpg6Ma_UEE3pYt2NQ?name=all-MiniLM-L6-v2.zip&oauth_token=TUhMOU1BV1gwWUE%3D%7C1695445530%7CNN8X0Y0SQ0NfJvMxNZCnumpJaurxCDaT%2FdK70Al%2Bgh0%3D&s=YkW8AVqTosiF"
```

Comment:
Is this link available outside of amazon? If no, just check in the model in order to run with ITs.